CHANGELOG.md

```diff
@@ -136,6 +136,7 @@
   libraries.
 
 ### Breaking Changes
+
```

Comment:
Did you mean to make a change log entry here? @tjprescott

Reply:
Originally yes, just to run the CI and verify it was failing.

Reply:
Oh this I assume was just a mistake cause its all the way back in beta.5